### ISSUE TITLE
Reproducing the results

### ISSUE THREAD

**Issue by stefanhacko** [zhengchuanpan/GMAN#31]
Hello,
Thank you very much for sharing your code with the community. 

After many attempts with different hyperparameters we have not been able to reproduce any results from the paper (or even get close). Was anyone been able to reproduce the results or do the authors have any pointers in how to achieve this?
Thank you.

**Comment by flowingrain** [comment]
Failed to reproduce as well

**Comment by zouguojian** [comment]
No, I can not reproduce the results from GMAN and DCRNN; why not? I do not know

**Comment by tarekbzahid** [comment]
Any ideas not being to get similar results after many trying different hyper-parameters
